addons/mail/static/tests/discuss/core/public_web/sub_channels.test.js

```diff
@@ -120,6 +120,25 @@ test("create sub thread from existing message (slow network)", async () => {
     });
 });
 
+test("create sub thread from deleted message", async () => {
```

Comment:
Would rename to _"Shows 'message is deleted' on sub-thread without name from empty message"_

This is longer, but not that much and it is very clear on what this does. No need to read internal code of test for the most part!